workflows/bacterial_genomics/quality_and_contamination_control/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [0.1] - 04-06-2024
```

Comment:
doesn't match release (1.0) stated in WF file

Reply:
Thanks !